istio/install/README.md

```diff
@@ -0,0 +1,16 @@
+# Istio manifest for Kubeflow
```

Comment:
Consider nesting this file path one layer deeper. We want to be able to extend this pattern for other large deps (Thinking of Knative here).
E.G. dependencies/istio

Reply:
done